### PR TITLE
Prioritize parcels according to fee per byte

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: core/src/miner/mem_pool.rs [hunk 1]
```diff
@@ -86,6 +86,8 @@ struct ParcelOrder {
     nonce_height: U256,
     /// Fee of the parcel.
     fee: U256,
+    /// Size of parcel's rlp serialized data
+    serialized_size: usize,
```
Comment (sgkim126) [inline core/src/miner/mem_pool.rs]: How about add `fee_per_bytes` field instaed of `serialized_size`, if it's only used to calculate `fee_per_byte`

Comment (majecty) [reply core/src/miner/mem_pool.rs]: 👍

**Comment by majecty** [comment]
@sgkim126 Fixed

File: core/src/miner/mem_pool.rs [hunk 2]
```diff
@@ -1239,4 +1248,65 @@ pub mod test {
 
         assert_eq!(fee + amount, item.cost());
     }
+
+    #[test]
+    fn test_fee_per_byte_order_simple() {
+        let order1 = create_parcel_order(U256::from(1000_000_000), 100);
+        let order2 = create_parcel_order(U256::from(1500_000_000), 200);
+        println!("{} {}", order1.fee_per_byte, order2.fee_per_byte);
```
Comment (kseo) [inline core/src/miner/mem_pool.rs]: Remove println here.

**Pull request merged**